docs/diagrams/FilterSequenceDiagram.puml

```diff
@@ -16,17 +16,17 @@ end box
 [-> LogicManager : execute("filter t/diabetes")
 activate LogicManager
 
-LogicManager -> AddressBookParser : parseCommand("filter t/diabetes")
-activate AddressBookParser
+LogicManager -> NurseyBookParser : parseCommand("filter t/diabetes")
+activate NurseyBookParser
 
 create FilterCommandParser
-AddressBookParser -> FilterCommandParser
+NurseyBookParser -> FilterCommandParser
 activate FilterCommandParser
 
-FilterCommandParser --> AddressBookParser
+FilterCommandParser --> NurseyBookParser
 deactivate FilterCommandParser
 
-AddressBookParser -> FilterCommandParser: parse("t/diabetes")
+NurseyBookParser -> FilterCommandParser: parse("t/diabetes")
 activate FilterCommandParser
 
 create ElderlyHasTagPredicate
```

Comment:
Not compulsory but I think it could be clearer if we passed in the tag into the ElderlyHasTagPredicate constructor and the predicate into the FilterCommand constructor? I'll leave it up to u since omission is allowed as well! (Picture is from tutorial 8 -> the person and tag constructors have arguments passed into them)
![Screenshot 2021-11-06 at 2 56 06 AM](https://user-images.githubusercontent.com/77181384/140564082-cbaa9dd5-7d8d-46fe-95b6-10255ceff838.png)

Reply:
Good suggestion! I'll change it!